src/components/plots/stackedAreaPlot.ts

```diff
@@ -85,8 +87,8 @@ export module Plots {
       if (scale == null) {
         return;
       }
-      scale.addPaddingException(this, 0);
-      scale.addIncludedValue(this, 0);
+      scale.addPaddingExceptionsProvider(this._baselineValueProvider);
+      scale.addIncludedValuesProvider(this._baselineValueProvider);
```

Comment:
Looks like this was a bug, although I don't think the baseline can be set on `StackedArea`? At least it doesn't seem like it would work correctly.